frontend/packages/operator-lifecycle-manager/src/plugin.tsx

```diff
@@ -281,6 +282,18 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [FLAGS.CAN_LIST_OPERATOR_GROUP],
     },
   },
+  {
+    type: 'Page/Resource/Details',
+    properties: {
+      model: ConsoleOperatorConfigModel,
+      loader: async () =>
+        (
+          await import(
+            './components/console-operator-config' /* webpackChunkName: "console-operator-config" */
+          )
+        ).ConsoleOperatorConfigDetailsPage,
+    },
+  },
```

Comment:
The console operator details page should really be part of the core console, although I understand why you put it here since the dialog is part of the OLM package. It would be better if we can move the details page into console-app and the modal into console-shared, although I don't know how much of a ripple effect that has :/

Reply:
Done.  Wasn't too bad.  https://github.com/openshift/console/pull/8919/commits/582fed15426e4d7a27912b762e314f141af8aee6